python/fledge/services/core/api/audit.py

```diff
@@ -114,14 +116,22 @@ async def create_audit_entry(request):
                    'severity': severity,
                    'details': details
                    }
-        return web.json_response(message)
     except AttributeError as e:
         # Return error for wrong severity method
         err_msg = "severity type {} is not supported".format(severity)
         _logger.error("Error in create_audit_entry(): %s | %s", err_msg, str(e))
-        raise web.HTTPNotFound(reason=err_msg)
+        raise web.HTTPNotFound(reason=err_msg, body=json.dumps({"error": err_msg}))
+    except StorageServerError as ex:
+        if int(ex.code) in range(400, 500):
+            err_msg = 'Audit entry cannot be logged. {}'.format(ex.error['message'])
+            raise web.HTTPBadRequest(body=json.dumps({"error": err_msg}))
+        else:
+            err_msg = 'Failed to log audit entry. {}'.format(ex.error['message'])
+            raise web.HTTPInternalServerError(body=json.dumps({"error": err_msg}))
```

Comment:
Add 'reason' consistency wise across API & for the key KV change (same as above)

Reply:
default reason text will be there? or we should add explicitly? As we are adding response body so I though we can skip custom reason text.

Reply:
reason text is only appears if we return explicitly (i.e reason="") - We had this style previously and GUI client mostly rely on this text if HTTP status code range lies between (3XX to 5XX).

We already had a JIRA for API improvements to return JSON response in body. https://scaledb.atlassian.net/browse/FOGL-4547